universitat-basel-iberoromanistik.csl

```diff
@@ -28,7 +28,7 @@
         <names variable="translator" delimiter="/" prefix="[" suffix="]">
           <label form="short" suffix=" "/>
           <name delimiter=" / " delimiter-precedes-last="always"/>
-          <et-al prefix=" (" suffix=") " font-style="italic"/>
+          <et-al font-style="italic"/>
```

Comment:
Do we need to redefine the term here as for the [] case  above?

Reply:
No, I checked the documentation and they don't actually put et al in parentheses